src/openrct2/interface/colour.c

```diff
@@ -38,13 +38,9 @@ enum
 
 void colours_init_maps()
 {
-    if (g1Elements == NULL) {
-        return;
-    }
-
     // Get colour maps from g1
     for (sint32 i = 0; i < 32; i++) {
-        rct_g1_element *g1Element = &g1Elements[SPR_PALETTE_2_START + i];
+        rct_g1_element *g1Element = gfx_get_g1_element(SPR_PALETTE_2_START + i);
```

Comment:
Can you add a `TODO` here to remove the sprite id addition?

Reply:
Why? The tag is called `SPR_PALETTE_2_START`, so `SPR_PALETTE_2_START + i` makes sense to me.